app/assets/stylesheets/common.scss

```diff
@@ -0,0 +1,111 @@
+@import "semantic-ui";
+
+// TODO: Replace with a local font
+@import url("//fonts.googleapis.com/css?family=Maven+Pro:700,900");
+
+html, body{
+  height: 100%;
+}
+
+.header {
+  font-family: "Maven Pro", $font-family;
+}
+
+.ui.pusher {
+  min-height: 100%;
+}
+
+.main.container, .main.grid {
+  margin-top: 5em;
+}
+
+.item.padder {
+  margin-top: 2em;
+}
+
+.ui.menu .item img.logo {
+  margin-right: 1.5em;
+}
+
+.ui.footer.segment {
+  margin: 5em 0em 0em;
+  padding: 5em 0em;
+}
+
+.ui.menu {
+  & > .hidden, & > .right.menu > .hidden {
+    display: none;
+  }
+}
+
+.clickable {
+  cursor: pointer;
+}
+
+// Semantic UI has these classes, however they're only applicable to
+// grids, containers, rows and columns.
+// plus, there isn't any `mobile hidden`, `X hidden` class.
+// this snippet is using the same class names and same approach
+// plus a bit more but to all elements.
+//
+// see https://github.com/Semantic-Org/Semantic-UI/issues/1114
+
+/* Mobile */
+@media only screen and (max-width: 767px) {
+  [class*="mobile hidden"],
+  [class*="tablet only"]:not(.mobile),
+  [class*="computer only"]:not(.mobile),
+  [class*="large screen only"]:not(.mobile),
+  [class*="widescreen only"]:not(.mobile),
+  [class*="or lower hidden"] {
+    display: none !important;
+  }
+}
+
+/* Tablet / iPad Portrait */
+@media only screen and (min-width: 768px) and (max-width: 991px) {
+  [class*="mobile only"]:not(.tablet),
+  [class*="tablet hidden"],
+  [class*="computer only"]:not(.tablet),
+  [class*="large screen only"]:not(.tablet),
+  [class*="widescreen only"]:not(.tablet),
+  [class*="or lower hidden"]:not(.mobile) {
+    display: none !important;
+  }
+}
+
+/* Computer / Desktop / iPad Landscape */
+@media only screen and (min-width: 992px) and (max-width: 1199px) {
+  [class*="mobile only"]:not(.computer),
+  [class*="tablet only"]:not(.computer),
+  [class*="computer hidden"],
+  [class*="large screen only"]:not(.computer),
+  [class*="widescreen only"]:not(.computer),
+  [class*="or lower hidden"]:not(.tablet):not(.mobile) {
+    display: none !important;
+  }
+}
+
+/* Large Monitor */
+@media only screen and (min-width: 1200px) and (max-width: 1919px) {
+  [class*="mobile only"]:not([class*="large screen"]),
+  [class*="tablet only"]:not([class*="large screen"]),
+  [class*="computer only"]:not([class*="large screen"]),
```

Comment:
such syntax, very highlight. is the file extension wrong? Should it be `sass` or something?

Reply:
No its just straight CSS, not SCSS